src/protocol-curr/xdr/Stellar-overlay.x

```diff
@@ -185,6 +187,16 @@ case SURVEY_TOPOLOGY:
     TopologyResponseBody topologyResponseBody;
 };
 
+struct FloodAdvert
+{
+    Hash txHashes<>;
```

Comment:
need to put some high upper bound on those arrays instead of just relying on max message size

Reply:
I added 1000 as the upper bound.

Reply:
I updated the code so that the upper bound is defined to be 1000 in the xdr file.